content/docs/getting-started/adding-a-catalog-item/roadie-cli/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Roadie CLI
+publishedDate: '2022-04-04T14:00:00.0Z'
+description: How to configure the Roadie CLI to create a catalog item.
+
+order: 4
+humanName: Roadie CLI
+examples:
+  - name: Component
+    language: bash
+    content: |
+      roadie-cli roadie-entity:create \
+        --kind component \
+        --name first-component \
+        --apiToken $ROADIE_API_TOKEN
+  - name: Resource
+    language: bash
+    content: |
+      roadie-cli roadie-entity:create \
+        --kind resource \
+        --name first-resource \
+        --apiToken $ROADIE_API_TOKEN
+  - name: System
+    language: bash
+    content: |
+      roadie-cli roadie-entity:create \
+        --kind system \
+        --name first-api \
+        --apiToken $ROADIE_API_TOKEN
+  - name: API
+    language: bash
+    content: |
+      roadie-cli roadie-entity:create \
+        --kind api \
+        --name first-api \
+        --apiToken $ROADIE_API_TOKEN
+  - name: Domain
+    language: bash
+    content: |
+      roadie-cli roadie-entity:create \
+        --kind domain \
+        --name first-api \
+        --apiToken $ROADIE_API_TOKEN
+---
+
+The Roadie Entity CLI allows you to create, update and delete entities in the Roadie catalog via the public API. This option allows you to manage entities from sources where Roadie does not have an existing Entity provider.
+
+### Get an API token
+
+Before you start you will first need to [get an API key](/docs/api/authorization/).
+
+### Install the CLI
+
+You can install the cli with the following command
+
+```bash
+npm install -g @roadiehq/roadie-cli
+```
+
+### Load a catalog item
```

Comment:
I need more info on what this is actually going to do. I can't tell if it's going to produce a YAML file for me that I then need to commit or if it's going to send a HTTP request to the Roadie API to create an entity in the catalog.

Reply:
Please let me know how it looks now.